### PR TITLE
Create journal-of-applied-pharmaceutical-research.csl

### PR DESCRIPTION
**Pull request by joapreditor**

### CODE REVIEW

**Comment by rmzelle** [comment]
@joapreditor, could you walk through https://github.com/citation-style-language/styles/wiki/Style-Requirements and make changes as required? Note that you can just update this pull request per the instructions given above (no need to create another new pull request).

File: journal-of-applied-pharmaceutical-research.csl [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+    <contributor>
+      <name>Sebastian Karcher</name>
+    </contributor>
+    <contributor>
+      <name>Sean Takats</name>
+      <email>stakats@gmu.edu</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <summary>Vancouver style as outlined by International Committee of Medical Journal Editors Uniform Requirements for Manuscripts Submitted to Biomedical Journals: Sample References</summary>
```
Comment (damnation333) [inline journal-of-applied-pharmaceutical-research.csl]: Did you write this? If not, adapt or remove.

File: journal-of-applied-pharmaceutical-research.csl [hunk 2]
```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+  <info>
+    <title>Journal of Applied Pharmaceutical Research</title>
+    <title-short>JOAPR</title-short>
+    <id>http://www.zotero.org/styles/journal-of-applied-pharmaceutical-research</id>
+    <link href="http://www.zotero.org/styles/journal-of-applied-pharmaceutical-research" rel="self"/>
+    <link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
```
Comment (damnation333) [inline journal-of-applied-pharmaceutical-research.csl]: can you ad a rel="template" with the style you used as a template?

Comment (rmzelle) [inline journal-of-applied-pharmaceutical-research.csl]: @joapreditor, see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: journal-of-applied-pharmaceutical-research.csl (hunk 1 above)
Comment (rmzelle) [inline journal-of-applied-pharmaceutical-research.csl]: @joapreditor see https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link

**Comment by joapreditor** [comment]
I wish to add reference style of Journal of Applied Pharmaceutical Research (JOAPR) to Mendeley so that when an author is preparing a manuscript, he/she can select citation style from mendeley software for JOAPR. In order to do so, I had generated a pull request # 2546 and submitted the reference style to repository. Then there were few queries and corrections. Since last 2 days there is no further queries or corrections were suggested.
I wish to know from you what I need to do next to add it to mendeley repository and this style be downloaded automatically with mendeley software.
I am new to this and doing it for the first time. Kindly bear with me

**Comment by adam3smith** [comment]
you're all set for right now. We need to do a final review and will then either have some additional requests or accept the style, which will then be available via Mendeley shortly

**Comment by rmzelle** [comment]
> I wish to know from you what I need to do next

Does this journal only publish in English? If so, the style locale should be fixed to either US or British English (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale). The "documentation" link also still needs to be updated (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link), as well as the style summary.

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**